docs/build/tools/netrunner.md

```diff
@@ -0,0 +1,189 @@
+
+That repository's README details the tool.
+
+Clone the repository with:
+
+```
+git clone https://github.com/ava-labs/avalanche-network-runner.git
+```
+
+Unless otherwise specified, file paths given below are relative to the root of this repository. 
+
+## Usage
+
+The basic pattern for using the Avalanche Network Runner is:
+
+* Define the network
+* Start the network
+* Interact with it
+* Shutdown
+
+The [api](https://github.com/ava-labs/avalanche-network-runner/tree/main/api) package contains client code that allows the user to make API calls to Avalanche nodes within the network.
+
+Please find examples of usage in the [examples](https://github.com/ava-labs/avalanche-network-runner/tree/main/examples) subdirectory.
+
+## Supported Backends
+
+As you can see in the README, a `Network` interface is exposed. There are two implementations of this interface that you may use. One is the [local](https://github.com/ava-labs/avalanche-network-runner/tree/main/local) implementation where each node runs in an isolated process on your machine. The other is the [Kubernetes](https://github.com/ava-labs/avalanche-network-runner/tree/main/k8s) implementation where each node runs in its own Kubernetes pod.
+
+## Local Backend
+
+The simplest and most straightforward way to use the Avalanche Network Runner is to use the local implementation. When you create a network with this tool, each node runs in a process on your machine. **Please note that the maximum size and performance of this network depend on your machine's specifications.**
+
+An example can be found at [examples/local/main.go](https://github.com/ava-labs/avalanche-network-runner/blob/main/examples/local/main.go). It is a simple setup which creates a network of 5 nodes, and performs some simple API calls.
+
+Creating a network is as simple as:
+
+```
+network, err := local.NewDefaultNetwork(log, binaryPath)
+```
+
+where `log` is a logger of type [logging.Logger](https://github.com/ava-labs/avalanchego/blob/master/utils/logging/logger.go#L12) and `binaryPath` is the path of the AvalancheGo binary that each node that exists on network startup will run.
+
+For example, the below snippet creates a new network using default configurations, and each node in the network runs the binaries at `/home/user/go/src/github.com/ava-labs/avalanchego/build`:
+
+```
+network, err := local.NewDefaultNetwork(log,"/home/user/go/src/github.com/ava-labs/avalanchego/build")
+```
+
+**Once you create a network, you must eventually call `Stop()` on it to make sure all of the nodes in the network stop. **Calling this method kills all of the Avalanche nodes in the network. You probably want to call this method in a `defer` statement to make sure it runs.
+
+To wait until the network is ready to use, use the network's `Healthy` method. It returns a channel which will be notified when all nodes are healthy.
+
+Each node has a unique name. Use the network's `GetNodeNames()` method to get the names of all nodes.
```

Comment:
The name of the methods is GetNodesNames. Probably GetNodeNames is better? Anyway the name is the former.

Reply:
You're right, although I believe that to be linguistically bad, but I am no native speaker so not sure.

Reply:
Me neither. In such a case we can change the method name.

Reply:
Changed in the library